exsclaim/journal.py

```diff
@@ -39,6 +39,7 @@ class JournalFamily():
     open_param =        "URL parameter optionally noting open results only"
     journal_param =     "URL paremter noting journal to search"
     date_range_prarm =  "URL parameter noting range of dates to search"
+    pub_type =          "URL paramter noting publication type (specific to Wiley)"
```

Comment:
Just make this an empty string for the base class, and describe it in a comment for now. (Or else the other journals will make their urls have "URL parameter noting..." in it